games-action/ddnet/ddnet-2023.03.09.recipe

```diff
@@ -0,0 +1,125 @@
+SUMMARY="A free cooperative platformer game based on Teeworlds"
+DESCRIPTION="DDraceNetwork (DDNet) is an actively maintained version of \
+DDRace, a Teeworlds modification with a unique cooperative gameplay. Help \
+each other play through custom maps with up to 64 players, compete against \
+the best in international tournaments, design your own maps, or run your own \
+server. The official servers are around the world. All ranks made on official \
+servers are available worldwide and you can collect points!"
+HOMEPAGE="https://ddnet.org"
+COPYRIGHT="2007-2014 Magnus Auvinen \
+	2010-2011 Shereef Marzouk \
+	2013-2022 Dennis Felsing"
+LICENSE="Zlib
+	CC-BY-SA-3.0
+	SIL Open Font License v1.1"
+REVISION="1"
+srcGitRev="398bc0d5ddff61d8315637ef6206d8a502acbe27"
+SOURCE_URI="https://github.com/ddnet/ddnet/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="46fa57a2ceaad64c2500c577bab75b00c1f68849c2dfe4a27ec0d80eb398ccf6"
+SOURCE_DIR="ddnet-$srcGitRev"
+ADDITIONAL_FILES="ddnet.rdef.in"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	ddnet$secondaryArchSuffix = $portVersion
+	app:DDNet = $portVersion
+	cmd:ddnet_srv = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libavcodec$secondaryArchSuffix
+	lib:libavformat$secondaryArchSuffix
+	lib:libavutil$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	lib:libcrypto$secondaryArchSuffix
+	lib:libfreetype$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libGLU$secondaryArchSuffix
+	lib:libogg$secondaryArchSuffix
+	lib:libopus$secondaryArchSuffix
+	lib:libopusfile$secondaryArchSuffix
+	lib:libpng16$secondaryArchSuffix
+	lib:libSDL2_2.0$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix
+	lib:libswscale$secondaryArchSuffix
+	lib:libswresample$secondaryArchSuffix
+	lib:libsqlite3$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libavcodec$secondaryArchSuffix
+	devel:libavformat$secondaryArchSuffix
+	devel:libavutil$secondaryArchSuffix
+	devel:libcurl$secondaryArchSuffix
+	devel:libcrypto$secondaryArchSuffix
+	devel:libfreetype$secondaryArchSuffix
+	devel:libGL$secondaryArchSuffix
+	devel:libGLU$secondaryArchSuffix
+	devel:libogg$secondaryArchSuffix
+	devel:libopus$secondaryArchSuffix
+	devel:libopusfile$secondaryArchSuffix
+	devel:libpng16$secondaryArchSuffix
+	devel:libSDL2_2.0$secondaryArchSuffix
+	devel:libssl$secondaryArchSuffix
+	devel:libsqlite3$secondaryArchSuffix
+	devel:libswscale$secondaryArchSuffix
+	devel:libswresample$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cargo$secondaryArchSuffix >= 0.61.0
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	cmd:ninja
+	"
+
+BUILD()
+{
+cmake -B build -S . \
+		-DCMAKE_INSTALL_PREFIX=$appsDir/DDNet \
```

Comment:
ddnet uses GNUInstallDirs: https://github.com/ddnet/ddnet/blob/master/CMakeLists.txt#L3020

Maybe worth to use $cmakeDirArgs

Reply:
Doesn't exactly work.